tests/models/models_test.py

```diff
@@ -155,8 +155,8 @@ def test_dptnet(fb):
 
 
 def test_dcunet():
-    _, istft = make_enc_dec("stft", 512, 512)
-    input_samples = istft(torch.zeros((514, 17))).shape[0]
+    _, istft = make_enc_dec("stft", 1024, 1024, 256, 16000)
+    input_samples = istft(torch.zeros((1026, 17))).shape[0]
```

Comment:
It took time to find out what the bug was because those tests were not clear enough. 
Let's try to improve it: 
- Pass everything to `make_enc_dec` as kwargs, this is unclear currently.
- Add a `n_fft` variable, and reference `n_fft + 2` in the `istft` to be clear.